src/llmq/quorums_chainlocks.cpp

```diff
@@ -301,18 +301,7 @@ void CChainLocksHandler::TrySignChainTip()
                 break;
             }
 
-            decltype(blockTxs.begin()->second) txids;
-            {
-                LOCK(cs);
-                auto it = blockTxs.find(pindexWalk->GetBlockHash());
-                if (it == blockTxs.end()) {
-                    // this should actually not happen as NewPoWValidBlock should have been called before
-                    LogPrint("chainlocks", "CChainLocksHandler::%s -- blockTxs for %s not found\n", __func__,
-                              pindexWalk->GetBlockHash().ToString());
-                    return;
-                }
-                txids = it->second;
-            }
+            auto txids = GetBlockTxs(pindexWalk->GetBlockHash());
```

Comment:
Should probably handle `nullptr` case to avoid undefined behaviour of `*txids` below.

Reply:
done :+1: